airflow/sensors/bash.py

```diff
@@ -40,6 +41,12 @@ class BashSensor(BaseSensorOperator):
         of inheriting the current process environment, which is the default
         behavior. (templated)
     :param output_encoding: output encoding of bash command.
+    :param retry_exit_code: If task exits with this code, treat the sensor
+        as not-yet-complete and retry the check later according to the
+        usual retry/timeout settings. Any other return code will be
+        treated as an error, and cause the sensor to file. If set to
+        ``None`` (the default), any non-zero exit code will cause a retry
+        and the task will never raise an error except on time-out.
```

Comment:
> Any other return code will be treated as an error, and cause the sensor to file.

This does not match the implementation, where `0` is always considered successful even if `retry_exit_code` is set to non-zero.

(Also there’s a typo in this sentence.)

Reply:
:face_palm:

Will fix.

Reply:
Classic case of "know what I mean, not what I've typed." Sorry about that. Fixed.